homeassistant/components/hdmi_cec/__init__.py

```diff
@@ -407,22 +404,22 @@ def _set_attr_name(self):
     def _hdmi_cec_unavailable(self, callback_event):
         # Change state to unavailable. Without this, entity would remain in
         # its last state, since the state changes are pushed.
-        self._state = STATE_UNAVAILABLE
+        self._player_state = None
```

Comment:
This changes the behavior from `unavailable` to `unknown`
Maybe it should instead implement the `available` property